client/src/components/Review Widget/ReviewSection.jsx

```diff
@@ -0,0 +1,21 @@
+import { useParams } from 'react-router-dom'
+import React from 'react';
+import ReviewList from './ReviewList.jsx'
+import ReviewMetaData from './ReviewMetaData.jsx'
+import './cssFiles/reviewSection.css'
+
+const ReviewSection = () => {
+  const { productId } = useParams()
+
+  return (
+    <div>
+      <h3>Ratings and Reviews</h3>
+      <div className='reviewSection'>
+        <ReviewMetaData className='metaData' product_id={productId} />
```

Comment:
The className property here on the component call for ReviewMetaData seems unnecessary. I don't see it being used as a prop inside ReviewMetaData and a className I don't believe can be applied here.

Reply:
You're totally right - this was left over from when i was wrestling with flexbox the first time around. Deleted in the new branch.